views/blocks/our-reporting-grid.twig

```diff
@@ -25,7 +25,11 @@
             >
                 <div class="our-reporting-card__main">
                     <span class="our-reporting-card__heading">{{ card.heading }}</span>
-                    <p class="our-reporting-card__strapline">{{ card.strapline }}</p>
+                    {% if card.content_type == strapline %}
+                        <p class="our-reporting-card__strapline">{{ card.strapline }}</p>
+                    {% elseif card.content_type == excerpt %}
+                        <p class="our-reporting-card__excerpt">{{card.excerpt}}</p>
+                    {% endif %}
```

Comment:
Two points here, can you make sure these values are strings, but also I don't think the values match up with the values in ACF? They're meant to be `card_strapline` etc?